src/fragments/lib/auth/js/emailpassword.mdx

```diff
@@ -141,6 +141,107 @@ async function signIn() {
 }
 ```
 
+The sign in flows can be as simple as just a user name and password as shown above, or multi-step process based on how Auth was configured. Amazon Cognito will return a user object when `Auth.signIn` is called that contains `challengeName` and `challengeParam`. Depending on what is enabled for each user, they may need to pass additional challenges such as multi-factor authentication (MFA), time-based one time password (TOTP), or a custom challenge such as CAPTCHA. 
+
+Based on the two parameters returned in the user object, you can then call the corresponding functions below:
+
+
+### Confirm sign in with SMS MFA
+
+If the user has SMS Multi-Factor Authentication (MFA) enabled, Cognito responds with a challenge that requires the user to provide an SMS verification code. This is where you can trigger `Auth.confirmSignIn()` and pass in the code provided by the user.  Amplify will then verify with Cognito that the SMS code is valid and complete the sign in process via Cognito returning access, ID, and refresh tokens that are all handled by Amplify internally.
+
+```javascript
+import { Auth } from 'aws-amplify';
+
+
+async function signInWithMFA() {
+  try {
+    const user = await Auth.signIn(username, password);
+    if (
+      user.challengeName === 'SMS_MFA' ||
+      user.challengeName === 'SOFTWARE_TOKEN_MFA'
+    ) {
+      // You need to get the code from the UI inputs
+      // and then trigger the following function with a button click.
+      const code = getCodeFromUserInput();
+      // If MFA is enabled, sign-in should be confirmed with the confirmation code
+      const loggedUser = await Auth.confirmSignIn(
+        user, // Return object from Auth.signIn()
+        code, // Confirmation code
+        mfaType // MFA Type e.g. SMS_MFA, SOFTWARE_TOKEN_MFA
```

Comment:
probably we can define this argument in a variable

Reply:
Updated with new commit.